drop_table_read.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
```

Comment:
could you reformat with e.g. `black` please?

Reply:
Done